sql3/test/defs/defs_date_functions.go

```diff
@@ -24,21 +24,21 @@ var datePartTests = TableTest{
 		{
 			name: "DatePartIncorrectParamsCount",
```

Comment:
Can you change the names as well.

Reply:
Fixed. I think the names got added in a different ticket; they weren't there when I was doing the original work.